gpkit/nomial_data.py

```diff
@@ -107,7 +107,7 @@ def diff(self, var):
     def __eq__(self, other):
         """Equality test"""
         if not all(hasattr(other, a) for a in ("exps", "cs", "units")):
-            return NotImplemented
+            return False
```

Comment:
I don't love this fix, because it prevents me from writing a class that doesn't have exps/cs/units, but knows how to compare itself to a nomial data.

I'm about to propose something else in another PR.